lib/rooms.rb

```diff
@@ -0,0 +1,28 @@
+# Room logic.  SIMPLE
+# I need this to provide me with 20 numbered rooms starting at the number 1 and ending at the number 20.
+# This also needs to (I believe) hold the room cost.  Each room has a $200 charge per night.
+
+module HotelHedwig
+  class Rooms
+
+    attr_reader :room_num, :room_rate
+
+# Initialize variables to access the room number AND the charge per room.  I believe this will make it easier to calculate in the end.
+
+    def initialize(room_num)
+      @room_num = room_num
+      @room_rate = 200
+    end
+
+# Using a self.all method to work out the room numbering system.
+
+    def self.all
```

Comment:
A good start on this, but you'd need to initialize number (probably to 0) and having `return rooms` at the end of the method.

Reply:
It is a good idea, down the road to have a `.all` method like this, but also notice that this method generates an all new list each time.